Context/SearchContextInterface.php

```diff
@@ -90,4 +90,11 @@ public function getFacetSortOrderProvider();
      * @return \Markup\NeedleBundle\Intercept\InterceptorInterface
      **/
     public function getInterceptor();
+
```

Comment:
(Especially as it's a breaking change - it'll make any class implementing this fatal unless it's updated.  This PR shouldn't change the behaviour, just make it more flexible for decorating the behaviour)